app/.snyk

```diff
@@ -4,40 +4,40 @@ version: v1.14.1
 ignore:
   SNYK-JS-LODASH-590103:
     - '@emotion/core > @emotion/css > babel-plugin-emotion > @babel/helper-module-imports > @babel/types > lodash':
-        reason: No upgrade available
-        expires: '2020-10-17T17:00:51.078Z'
+        reason: No remediations available
```

Comment:
I feel synk has become noisy and should be removed as blocker for our PRs.

Reply:
Agree. We have processes to check it regularly, so I believe it's best to change to to warn us instead of block us.

Reply:
this will be done in a separate pr